acceptance/api/v1/namespace_auth_test.go

```diff
@@ -190,9 +190,9 @@ var _ = Describe("Users Namespace", LNamespace, func() {
 					response.Body.Close()
 				})
 
-				It("doesn't show the other user's namespace", func() {
+				It("shows the other user's namespace", func() {
 					response := showNamespace(user1, passwordUser1, commonNamespace)
-					Expect(response.StatusCode).To(Equal(http.StatusForbidden))
+					Expect(response.StatusCode).To(Equal(http.StatusOK))
```

Comment:
We still have it but I had to change this because of a small change in the actual behavior. This was the difference that I documented in the description, and it's basically due because we are not cleaning up the roles of deleted namespaces.
I'm not sure if this should be the desired behavior or not. When using OIDC auth the roles will be added anyway. Recreation of a namespace with the same name will keep the same roles assigned. Maybe we want this, or maybe not.

I'm thinking that is probably better to cleanup the roles.
Regarding the OIDC auth maybe we should not even assign namespace-scoped roles if the namespace doesn't exists.

WDYT?

These are the existing tests about the namespaces:

https://github.com/epinio/epinio/blob/04de74d4ef482e561d65292c9c515e7c46abcba6/acceptance/api/v1/namespace_auth_test.go#L109-L119

https://github.com/epinio/epinio/blob/04de74d4ef482e561d65292c9c515e7c46abcba6/acceptance/api/v1/namespace_auth_test.go#L122-L135

Reply:
Base also on the latest discussion I've added the cleanup of the old namescoped roles. When a namespace is deleted also the user roles that were bound to that namespace are deleted. I can revert the test and keep the original behavior. :slightly_smiling_face: